nmigen/test/test_hdl_ast.py

```diff
@@ -11,13 +11,13 @@ def test_wrap(self):
         self.assertIsInstance(Value.wrap(True), Const)
         c = Const(0)
         self.assertIs(Value.wrap(c), c)
-        with self.assertRaises(TypeError,
-                msg="Object ''str'' is not an nMigen value"):
+        with self.assertRaises(
+                TypeError, msg="Object ''str'' is not an nMigen value"):
```

Comment:
I would prefer that you not reformat an entire file in an unrelated commit.

Reply:
Done.